src/App.tsx

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
-import TrafficLight from './Components/TrafficLight';
-import './App.css';
+import React, { useEffect, useState } from "react";
+import TrafficLight from "./Components/TrafficLight";
+import "./App.css";
+
+const lightDurations = [1000, 2000, 4000];
 
 function App() {
+  const [activeColor, setActiveColor] = useState<number>(0);
```

Comment:
How about naming `firstLightColor`, and `secondLightColor`? That might improve the readability a lot.